src/commands/farm.rs

```diff
@@ -64,7 +64,10 @@ pub(crate) async fn farm(is_verbose: bool) -> Result<(Farmer, Node, SingleInstan
             .map_ok(|SyncingProgress { at, target, status: _ }| (target as _, at as _))
             .map_err(|err| eyre!("Sync failed because: {err}"));
 
-        let (target_block, current_block) = syncing_progress.next().await.unwrap()?;
+        let Some(syncing_result) = syncing_progress.next().await else {
+            return Err(eyre!("backoff mechanism timed out for node syncing on the SDK side"));
```

Comment:
This shouldn't be an error. We can be synced and therefore sdk sends nothing and closes stream. We should just skip progress bar and go to plotting

Reply:
oh I see, nice then! fixing